modules/runner/src/integrationTest/sql/insert/test_insert_type.test

```diff
@@ -1,7 +1,6 @@
 # name: test/sql/insert/test_insert_type.test
 # description: Test insert into from wrong type
 # group: [insert]
-# Ignored: https://issues.apache.org/jira/browse/IGNITE-18365
```

Comment:
does it mean IGNITE-18365 is not valid anymore?

Reply:
yes, will be closed just after merge the PR ( I suppose it will mean 'common agree that is true')